contrib/scrooge/src/python/pants/contrib/scrooge/tasks/thrift_linter_task.py

```diff
@@ -48,6 +48,9 @@ def register_options(cls, register):
              removal_hint="Use `--scrooge-linter-strict-default` instead",
              help='Sets the default strictness for targets. The `strict` option overrides '
                   'this value if it is set.')
+    register('--ignore-errors', default=False, advanced=True, type=bool,
```

Comment:
Given that this flag is new, what usecase are you anticipating for it? Completely ignoring errors feels like it's close to `--skip`, but will still render warnings? Are we expecting that folks will set this at the repository level, or is this being added for completeness? 

It might be good to turn your bulletpoints in the description into a table of inputs and outputs/expected behaviour.

Reply:
> Completely ignoring errors feels like it's close to --skip, but will still render warnings?

Correct, warnings are still rendered.

Updated the description with before vs after.